content/resources/tutorials/_index.md

```diff
@@ -8,4 +8,33 @@ TODO: [GH 124](https://github.com/ReproNim/repronim.org/issues/124)
 
 To get started with ReproNim, we have created a set of  tutorials to show how ReproNim’s tools and services support best practices laid out in [ReproNim’s principles of Reproducible Neuroimaging](/about/in-practice/) and [ReproNim’s 4 Core Actions](/about/in-practice/).  We have organized these tutorials based on our [user personas](/resources/getting-started/) to illustrate how ReproNim’s tools can meet the needs of different types of users with different skill sets and objectives. These tutorials are designed in a modular fashion, showing how individual steps to improve reproducibility can be fit together into an [overall reproducible workflow](/about/repronim-approach/).  
 
-Tutorials are organized into basic and more advanced approaches. Each tutorial is organized in a similar fashion,  providing a list of tools, necessary skills and system requirements, along with step by step instructions to implement and use the tools.    
+Tutorials are organized into basic and more advanced approaches. Each tutorial is organized in a similar fashion,  providing a list of tools, necessary skills and system requirements, along with step by step instructions to implement and use the tools. Below we provide a list of available tutorials, organized according to [ReproNim’s principles of Reproducible Neuroimaging](/about/in-practice/):
+
+**Principle 1:  Study Planning**
+
+* Creating a [neuroimaging data management and sharing plan](/resources/tutorials/data-management-and-sharing/)
+* Set up a data collection and annotation framework for a multisite study using [ReproSchema](https://repronim.netlify.app/resources/tutorials/reproschema/).
+
+**Principle 2: Data and metadata management**
+
+      * Actions: Standards, Annotation and Provenance, Version control 
+
+* *Basic:*  Using ReproNim tools to [convert data to the BIDS standard](/resources/tutorials/dicom-to-bids/) and [create a data dictionary](/resources/tutorials/data-dictionary/)
+* *Basic+:*  Adding standards and semantics to data dictionaries to promote data harmonization and findability
+* *Advanced:* Putting your derived data into BIDS derivatives and creating a semantically-enriched  data dictionary using NIDM, a standard for annotating neuroimaging data
+* *Advanced:* [DataLad containers/run \+, YODA principles](/resources/tutorials/repronim-containers/)
```

Comment:
Datalad containers/run + YODA, I think we don't need a comma.

Same for the next reference to the same tutorial.